vosekast_control/MQTT.py

```diff
@@ -37,10 +37,13 @@ def __init__(self, host):
         try:
             await self.client.connect(self.host)
         except ConnectionRefusedError:
+
             await self.connection_refused()
                 
     async def connection_refused(self):
-        self.logger.warning("Connection refused. Is the MQTT broker installed? Retrying.")
+        self.logger.warning(
+          "Connection refused. Is the MQTT broker installed? Retrying.")
```

Comment:
Maybe better `Is the MQTT Broker accessible?`

Reply:
ok